covalent/_results_manager/results_manager.py

```diff
@@ -133,50 +133,38 @@ def dummy_function(x):
     return result
 
 
-def _get_result_from_db(
-    db: DataStore, dispatch_id: str, wait: bool = False, status_only: bool = False
-) -> Result:
+def _get_result_from_dispatcher(
+    dispatch_id: str,
+    wait: bool = False,
+    dispatcher: str = get_config("dispatcher.address") + ":" + str(get_config("dispatcher.port")),
+    status_only: bool = False,
+) -> Dict:
 
     """
-    Internal function to get the results of a dispatch from a file without checking if it is ready to read.
+    Internal function to get the results of a dispatch from the server without checking if it is ready to read.
 
     Args:
-        db: The DataStore object which yields the SQLAlchemy session that stores results.
         dispatch_id: The dispatch id of the result.
         wait: Whether to wait for the result to be completed/failed, default is False.
+        status_only: If true, only returns result status, not the full result object, default is False.
+        dispatcher: Dispatcher server address, defaults to the address set in covalent.config.
 
     Returns:
-        result_object: The result object constructed from the DB lattice record.
+        The result object from the server.
 
     Raises:
         MissingLatticeRecordError: If the result is not found.
     """
-
-    while True:
-        with Session(db.engine) as session:
-            lattice_record = (
-                session.query(Lattice).where(Lattice.dispatch_id == dispatch_id).first()
-            )
-        try:
-            if not lattice_record:
-                raise MissingLatticeRecordError
-            elif not wait:
-                return lattice_record.status if status_only else result_from(lattice_record)
-            elif lattice_record.status in [
-                str(Result.COMPLETED),
-                str(Result.FAILED),
-                str(Result.CANCELLED),
-                str(Result.POSTPROCESSING_FAILED),
-                str(Result.PENDING_POSTPROCESSING),
-            ]:
-                return lattice_record.status if status_only else result_from(lattice_record)
-
-        except (FileNotFoundError, EOFError, _pickle.UnpicklingError):
-            if wait:
-                continue
-            raise RuntimeError(
-                "Result not ready to read yet. Please wait for a couple of seconds."
-            )
+    adapter = HTTPAdapter()
```

Comment:
should add a `max_retries=<something>` here to automate retries?

Reply:
I think retries are enabled automatically, even without `max_retries`?

Reply:
Yes, I think retries are automatic. To set more custom limits on retry: https://stackoverflow.com/questions/15431044/can-i-set-max-retries-for-requests-request

Reply:
I recommend opening another PR if we need to make enhancements. Resolving and merging this PR ASAP.